tests/reg_tests/commonUtils.py

```diff
@@ -248,6 +253,282 @@ def adjoint_test(handler, CFDSolver, ap):
     handler.root_print('Eval Functions Sens:')
     handler.root_add_dict(funcsSens, 1e-10, 1e-10)
 
+def assert_adjoint_sens_allclose(handler, CFDSolver):
```

Comment:
Would it be possible to add atol and rtol as optional arguments to each of these functions? I think that would be useful to have the option instead of hard-coding everything as 1e-10.